data/campaigns/Winds_of_Fate/units/Gates.cfg

```diff
@@ -0,0 +1,64 @@
+#textdomain wesnoth-wof
+
+[movetype]
+    name=gate
+    [movement_costs]
```

Comment:
What is the reason for these 1-point movement_costs?  Asking more out of curiosity than anything.

Reply:
After some testing, there does not seem to be any reason; it must have been an experimental hack attempting to solve one issue or another with the gates, early on. Removed these.